test/run_test.py

```diff
@@ -1122,21 +1098,19 @@ def load_specified_test_cases(filename: str) -> None:
     with open(filename, mode='r', encoding="utf-8-sig") as csv_file:
         csv_reader = csv.DictReader(csv_file)
         line_count = 0
-        global SPECIFIED_TEST_CASES_DICT
+        global SPECIFIED_TEST_MODULES
         for row in csv_reader:
             line_count += 1
             if line_count == 1:
                 if 'test_filename' not in row or 'test_case_name' not in row:
                     print('Data is missing necessary columns for test specification. Proceeding with default behavior.')
                     return
             test_filename = row['test_filename']
-            test_case_name = row['test_case_name']
             if test_filename not in TESTS:
                 print(f'Specified test_filename {test_filename} not found in TESTS. Skipping.')
                 continue
-            if test_filename not in SPECIFIED_TEST_CASES_DICT:
-                SPECIFIED_TEST_CASES_DICT[test_filename] = []
-            SPECIFIED_TEST_CASES_DICT[test_filename].append(test_case_name)
+            if test_filename not in SPECIFIED_TEST_MODULES:
```

Comment:
Do you think it could be worth it to make SPECIFIED_TEST_MODULES a set instead of a list for faster runtime here? Or would the runtime probably not matter since we have ~150 test modules?